dagshub/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.2.9"
+__version__ = "0.2.9.1"
```

Comment:
Don't add additional dots. Looked at what we had previously in our releases and it should be like this:
```suggestion
__version__ = "0.2.9v2"
```

Reply:
Also in general we shouldn't be bumping version numbers in PRs because they aren't always merged synchronously